apps/src/templates/VerticalImageResourceCard.jsx

```diff
@@ -180,16 +175,7 @@ const styles = {
     marginTop: 5,
     padding: 5,
     width: 258
-  },
-  ltr: {
-    float: 'left'
-  },
-  rtl: {
-    float: 'right',
-    textAlign: 'right'
```

Comment:
I'm having trouble telling from the screenshots what is changing visually as a result of this PR, and I won't have time to dig in until after a long block of interviews and meetings. deferring the work to fix the float problem seems ok to me, as long as this PR is not changing the existing behavior (or as long as it is not making it worse).

Also, I'm a bit rusty on RTL stuff but IIRC the floats could be replaced with flex styles to make them RTL-friendly. Not sure how much work that is to figure out. I would only suggest deferring the work if it is a substantial amount of work and we're confident this change doesn't leave us in a bad state in the interim.

Reply:
I've added more screenshots from production––hope that helps. I didn't see any difference, but could be missing something

Reply:
i tested this component locally as well on /congrats in LTR and RTL and everything looks good in chrome.

dave, i think the reason this "just works" without flex styling is because the title and description content are text nodes, so the browser will automatically reflow the content appropriately when we have `<html dir={ltr|rtl}>` at the root.

that being said, we should make sure this doesn't cause issues in other browsers -- i think it should be fine, but better to double check!

Reply:
Thanks, Maddie! It did seem weird we could just remove code without anything else, but what you're saying makes sense. I'll check locally too in safari and firefox

Reply:
haha, yes, I think this is what set off my spidey sense -- how could it still work if you just remove styles? but you both have said it, the browser's default rendering system takes care of it 👍

Reply:
Just checked on Safari and Firefox! Looks good.